### PR TITLE
Refactor duplicated function expressions for enabling and disabling a…

### PR DESCRIPTION
**Pull request by kencheung**
… user

### CODE REVIEW

**Comment by dhcole** [comment]
Thanks for submitting this @kencheung. We'll review it for our next release.

File: api/controllers/UserController.js [hunk 1]
```diff
@@ -258,16 +258,20 @@ module.exports = {
     });
   },
 
-  // Enable a disabled user
-  enable: function (req, res) {
-    // policies will ensure only admins can run this function
-    User.findOneById(req.route.params.id, function (err, user) {
+  findUserById: function(res, err, user, userDisabled, saveErrMsg) {
```
Comment (jeremiak) [inline api/controllers/UserController.js]: Thanks for submitting this PR, refactoring is always appreciated :smile: 

In this case, I'm not sure I'm sold on introducing a function that takes five positional arguments. I find it hard to remember a function who's signature includes five arguments and know what the order is. Instead, how about submitting some of these as keys on an object that is passed in. Something similar to what you see is the function signature for `res.send()` on line 265.

By way of example, `findUserById` might take something like `(res, err, { user: user, userDisabled: false, errMsg: ''})`.

Thoughts?

Comment (kencheung) [inline api/controllers/UserController.js]: That improves readability. I have revised the PR as suggested. See if it looks better to you.

**Comment by jeremiak** [comment]
Hi @kencheung - thanks for this and for making some of the changes I suggested.

Can you get this branch to pass tests? You should be able to run them locally with `npm run test` and have it exit successfully. As you can see above, Travis is reporting that this branch fails the tests.

**Comment by kencheung** [comment]
You are welcome. I have made the branch to pass tests as suggested.

**Comment by dhcole** [comment]
Thanks @kencheung! :ship:

**Pull request merged**